cves/CVE-2019-6975.yml

```diff
@@ -275,10 +305,10 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 400
+CWE_note: Excessive memory usage is a lack of control on resources.
```

Comment:
Agreed, did your research reveal any other checks in place to catch a run away memory use issue such as this?

Reply:
I did not see any checks for excessive memory usage. The new unit tests just add tests for these large numbers. And I do not know how the issue was discovered.